blazor-wasm/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Blazor.Hosting;
+
+namespace WebApplication1
```

Comment:
It would be clearer if the project's name matches the language of an example
```suggestion
namespace BlazorWasmExample
```